src/components/CodeView/index.spec.tsx

```diff
@@ -65,4 +100,72 @@ describe(__filename, () => {
     );
     expect(root.find(`.${styles.code}`)).toHaveLength(contentLines.length);
   });
+
+  it('renders an HTML ID for each line', () => {
+    const root = render({ content: 'line 1\nline 2' });
+
+    expect(root.find(`.${styles.line}`)).toHaveLength(2);
+    expect(root.find(`.${styles.line}`).at(0)).toHaveProp('id', 'L1');
+    expect(root.find(`.${styles.line}`).at(1)).toHaveProp('id', 'L2');
+  });
+
+  it('marks a row as selected', () => {
+    const selectedLine = 2;
+    const location = createFakeLocation({ hash: `#L${selectedLine}` });
+
+    const root = render({ content: 'line 1\nline 2', location });
+
+    expect(root.find(`.${styles.selectedLine}`)).toHaveLength(1);
```

Comment:
Can this check that exactly line 2 is selected?

Reply:
yep